helm/hpcc/templates/dfuserver.yaml

```diff
@@ -0,0 +1,63 @@
+{{- define "hpcc.dfuServerConfigMap" -}}
+apiVersion: v1
+metadata:
+  name: {{ .me.name }}-configmap
+data:
+  {{ .me.name }}.yaml: |
+    version: 1.0
+    dfuserver:
+{{ toYaml (omit .me "logging") | indent 6 }}
+{{- include "hpcc.generateLoggingConfig" . | indent 6 }}
+    global:
+{{ include "hpcc.generateGlobalConfigMap" .root | indent 6 }}
+{{- end -}}
+
+{{ range $.Values.dfuserver -}}
+{{- if not .disabled -}}
+{{- $secretsCategories := list "system" "storage" -}}
+{{- $commonCtx := dict "root" $ "me" . "secretsCategories" $secretsCategories "includeLabels" (list "lz") }}
+{{- $configSHA := include "hpcc.dfuServerConfigMap" $commonCtx | sha256sum }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .name | quote }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: {{ .name | quote }}
+  template:
+    metadata:
+      labels:
+        run: {{ .name | quote }}
+        accessDali: "yes"
+      annotations:
+        checksum/config: {{ $configSHA }}
+    spec:
+      serviceAccountName: "hpcc-default"
+      initContainers:
+        {{- include "hpcc.checkDataMount" $commonCtx | indent 6 }}
+      containers:
+      - name: {{ .name | quote }}
+        workingDir: /var/lib/HPCCSystems
+        command: [ dfuserver ]
+        args: [
+                {{ include "hpcc.configArg" . }},
+                {{ include "hpcc.daliArg" $ }}
+          ]
+{{ include "hpcc.addSentinelProbes" . | indent 8 }}
+{{- include "hpcc.addSecurityContext" $commonCtx | indent 8 }}
+{{- include "hpcc.addResources" (dict "me" .resources) | indent 8 }}
+{{ include "hpcc.addImageAttrs" $commonCtx | indent 8 }}
+        volumeMounts:
+{{ include "hpcc.addDataVolumeMount" $commonCtx | indent 8 }}
```

Comment:
dfuserver calls addDataVolumeMount .. passing "lz"
I think it would be clearer if addDataVolumeMount called addVoumeMounts with explicit "data" only
and others that wanted different mounts called addVolumeMounts not addData*